### PR TITLE
Add ABN and slogan on the footer area catered for responsive design.

### PR DESCRIPTION
**Pull request by Wongad**

### CODE REVIEW

File: css/layouts/responsive.css [hunk 1]
```diff
@@ -80,6 +80,58 @@
...
+  /* The footer abn and tagline */
+  .region-footer .block {
+    width: 50%;
+  }
+  .region-footer .menu::after {
+    position: relative;
+    float: left;
+    clear: both;
+    display: block;
+    content: "ABN 63 384 330 717";
```
Comment (typhonius) [inline css/layouts/responsive.css]: I'd be tempted to put the ABN in a template file. Consider looking at region--footer.tpl.php for this. You will be able to copy region--footer.tpl.php from the zen theme into this theme as a starter.

File: css/layouts/responsive.css [hunk 2]
```diff
@@ -80,6 +80,58 @@
...
+  .region-footer div.last::after {
+    position: absolute;
+    right: 4%;
+    margin-top: 2em;
+    text-align: right;
+    font-family: Georgia, Times, Times New Roman, serif;
+    font-style: italic;
+    font-size: 110%;
+    width: 30%;
+    content: "Financial outcomes for consumers, business and the community.";
```
Comment (typhonius) [inline css/layouts/responsive.css]: Same for this as above with the ABN.

File: css/layouts/responsive.css [hunk 3]
```diff
@@ -80,6 +80,58 @@
...
+  }
+  .region-footer div.last::after {
+    position: relative;
+    float: right;
+    clear: both;
+    width: 100%;
+    font-family: Georgia, Times, Times New Roman, serif;
+    font-style: italic;
+    font-size: 110%;
+    content: "Financial outcomes for consumers, business and the community.";
```
Comment (typhonius) [inline css/layouts/responsive.css]: Same for this as above with the ABN.

File: css/layouts/responsive.css (hunk 1 above)
Comment (gollyg) [inline css/layouts/responsive.css]: Yep, I agree. This is definitely content that should be output as HTML, not added in at the style layer. With these kind of decisions, you always need to assume things like CSS pseudo elements _won't_ be supported.

File: css/components/misc.css [hunk 4]
```diff
@@ -159,6 +159,9 @@
   padding: 0.2em 0.7em;
   border-left: 1px solid #FFFFFF;
 }
+.region-footer .block em {
+  font-family: Georgia, Times, Time New Roman, serif;
+}
```
Comment (gollyg) [inline css/components/misc.css:164]: This may be required, but seems to be a very specific implementation for a font face?

**Comment by gollyg** [comment]
@Wongad Needs to be rebased against current head.

**Pull request merged**